samples/display/display.rst

```diff
@@ -0,0 +1,10 @@
+.. _display-samples:
```

Comment:
you'll also need to edit samples/samples.rst to add this new display.rst to the index.